alacritty_terminal/src/term/mod.rs

```diff
@@ -1,9 +1,8 @@
 //! Exports the `Term` type which is a high-level API for the Grid.
```

Comment:
I'm skipping my review of this file until later. I don't think any of my suggestions should have an effect on this anyway and I don't want to rush it or delay the review until I'm done.

The only thing I can already say for sure is that `DamagedLines` is a terrible name for what it appears to be doing based on what I've read elsewhere.

Reply:
Yeah, I have an idea how to do it better. You could just wait and I push updates soon-ish.